jhputnam/mailroom.py

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env python3
+
+"""Mailroom script to send thank you emails to donors"""
+
+from operator import itemgetter
+
+DONORS = {"Derp McDerpston": [250.00, 650.00], "Billy Bob": [0.02],
+          "Cthulhu": [1000.00, 25000.0, 123456.0],
+          "Kayaba Akihiko": [10000.00, 3853.00, 6147.00],
+          "Doctor Evil": [1000000.00]}
+
```

Comment:
Convention for listing a dictionary should look like this with a single item on each line.  Makes it a little bit more readable:
`DONORS = {
    "Derp McDerpston": [250.00, 650.00],
    "Billy Bob": [0.02],
    "Cthulhu": [1000.00, 25000.0, 123456.0],
    "Kayaba Akihiko": [10000.00, 3853.00, 6147.00],
    "Doctor Evil": [1000000.00]
}`